### PR TITLE
Create dedicated page for validation checklist

### PR DESCRIPTION
**Pull request by Quetzacoalt91**
With this PR, both reviewed and contributors will now what is checked when a module is submitted on the marketplace.

### CODE REVIEW

File: src/content/1.7/development/database/db.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
+=========
+Dedicated page for running SQL request with DB Class. Rewrite me
+=========
```
Comment (Quetzacoalt91) [inline src/content/1.7/development/database/db.md]: ToDo

**Comment by Quetzacoalt91** [comment]
Removed `WIP` flag, now ready for review

File: src/content/1.7/development/database/db.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
+---
+title: Db class
+weight: 1
+---
+
+# Using the database with the Db class
```
Comment (apacios) [inline src/content/1.7/development/database/db.md]: Can we specify that in `executeS`, `getRow`, `getValue` and `execute` we need the DB_PREFIX ? 
In `update`, `delete`, `insert` we don't need to specify the prefix

Comment (Quetzacoalt91) [reply src/content/1.7/development/database/db.md]: done

**Pull request merged**